Lib/test/test_httplib.py

```diff
@@ -2021,13 +2005,6 @@ def test_tls13_pha(self):
         self.assertIs(h._context, context)
         self.assertFalse(h._context.post_handshake_auth)
 
-        with warnings.catch_warnings():
-            warnings.filterwarnings('ignore', 'key_file, cert_file and check_hostname are deprecated',
-                                    DeprecationWarning)
-            h = client.HTTPSConnection('localhost', 443, context=context,
-                                       cert_file=CERT_localhost)
-        self.assertTrue(h._context.post_handshake_auth)
-
```

Comment:
Should not it be an error?

Reply:
I updated the test rather than removing it.